src/pages/api/v1/prep-yatra/recruiter/index.ts

```diff
@@ -109,6 +113,36 @@ const handleAddRecruiter = async (
       );
     }
 
+    // Trigger gamification for recruiter addition
+    try {
+      let { data: gamificationData, error: gamificationError } = await updateUserPointsInDB(userId, 'RECRUITER_ADDED');
+      
+      // If user not found, create gamification record first
+      if (gamificationError && gamificationError === 'User not found') {
+        console.log('Creating gamification record for user:', userId);
+        const { data: newGamificationData, error: createError } = await addGamificationDocInDB(userId);
+        
```

Comment:
We don't need this. We've one API which we can call as Function. We can directly pass EVENT name and kit automatically can do the rest. 

Check existing code.